pyproject.toml

```diff
@@ -24,15 +24,18 @@ classifiers = [
 ]
 requires-python = ">=3.8"
 dependencies = [
-    "Scrapy >= 2.7.0",
+    "Scrapy >= 2.11.0",
     "url-matcher >= 0.5.0",
-    "w3lib >= 1.22.0",
+    "w3lib >= 2.2.0",
```

Comment:
Why are these increases of minimum supported versions necessary?

Reply:
Similar to https://github.com/zytedata/duplicate-url-discarder/pull/16#discussion_r1666707587